docs/administration-guide/databases/google-analytics.md

```diff
@@ -0,0 +1,73 @@
+## Working with Google Analytics in Metabase
+
+This page provides information on how to create and manage a connection to a [Google Analytics](https://cloud.google.com/analytics) dataset.
+
+## Prerequisites
+
+You will need to have a [Google Cloud Platform](https://cloud.google.com/) account and create the [project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project) you would like to use in Metabase. Consult the Google Cloud Platform documentation on how to [create and manage a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects) if you do not have one.
+
+## Google Cloud Platform: creating OAuth client ID
+
+To get the **Client ID** and **Client Secret**, follow Google's Cloud Platform Help on [how to create an OAuth client ID](https://support.google.com/cloud/answer/6158849?hl=en). For application type, select "Desktop App" (this avoids a `redirect_uri_mismatch` when requesting **Auth Codes**).
+
+### Enabling Google Analytics API
+To enable the Google Analytics API, see [Enable and disable APIs](https://support.google.com/googleapi/answer/6158841?hl=en).
```

Comment:
I just followed your instructions and had some difficulties at this step. Until I actually added the database in Metabase and came cross this help text with the _correct_ link to enable the GA API.

> To use Metabase with this data you must enable API access in the Google Developers Console. [Click here](https://console.developers.google.com/apis/api/analytics.googleapis.com/overview) to go to the console if you haven't already done so.

![image](https://user-images.githubusercontent.com/4390468/114425941-d7feff00-9bb9-11eb-96fe-1bccbae9e4ae.png)

Reply:
```suggestion
To enable the Google Analytics API, go to this page in the Google Cloud Platform console: https://console.cloud.google.com/apis/api/analytics.googleapis.com/overview. Double check that the previously created project is selected and click on "Enable".

For further documentation please refer to [Enable and disable APIs](https://support.google.com/googleapi/answer/6158841?hl=en).
```

Reply:
I see your point. This certainly improves the flow when following the instructions. Thanks for performing this correction.